src/Generators/SeederGenerator.php

```diff
@@ -24,14 +24,21 @@ public function generate()
         $this->checkConfigs();
 
         if (!file_exists($this->databaseSeederPath)) {
+            $databaseSeederPathSegments = explode('/', $this->databaseSeederPath);
+            array_pop($databaseSeederPathSegments);
+            $databaseSeederDir = implode('/', $databaseSeederPathSegments);
```

Comment:
please use helper for it https://github.com/RonasIT/laravel-helpers/blob/master/src/helpers.php#L395